src/AddIns/Uno.UI.Lottie/LottieVisualSourceProvider.cs

```diff
@@ -7,7 +7,7 @@
 #pragma warning disable 105
 // Keep the space as this will not be replaced by the
 // WinUI upgrade tool.
-using Microsoft. UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Controls;
```

Comment:
@jeromelaban Can I have more info about the problem that's being worked-around with the extra space here?

Reply:
> What was the error caused by the removal?

I don't recall exactly. It's some type not found or something similar.
Anyway, I adopted adding a comment instead of the space, so this satisfies the formatting and not cause other issues.